apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/BlockchainProcessorImpl.java

```diff
@@ -321,6 +332,10 @@ public BlockchainProcessorImpl(BlockValidator validator, javax.enterprise.event.
         this.taskDispatchManager = taskDispatchManager;
         this.txEvent = txEvent;
 
+        if (propertiesHolder.getBooleanProperty("apl.enablePublicKeyCache")) {
```

Comment:
You can inject specific cache itself instead of cacheManager

Reply:
It's a single way to initialize cache from Manager. There is not another approach to do cache initialization now.

Reply:
Cache is removed